cloudconfig/cloudinit/cloudinit_ubuntu.go

```diff
@@ -291,11 +309,13 @@ func (cfg *ubuntuCloudConfig) updateProxySettings(proxyCfg PackageManagerProxyCo
 			filename))
 	}
 
+	once := sync.Once{}
```

Comment:
wouldn't a bool be better suited to this?

```
once := false
addWaitSnapSeeded := func() {
    if once {
         return
    }
    cfg.AddRunCmd(waitSnapSeeded)
    once = true
}
```

Reply:
just that sync.Once is a bit heavy for something not requiring synchronisation?

Reply:
i considered doing it this way also but not sure why I went with sync.Once. i'll change to the func approach